inspectit-oce-core/src/main/resources/config/default.yml

```diff
@@ -131,10 +131,17 @@ inspectit:
           sinceNanos: long
         value: "new Double( (System.nanoTime() - sinceNanos) * 1E-6)"
 
+      get_method_fqn:
+        input:
+          methodName: String
+          clazz: Class
+        value: new StringBuilder(clazz.getName()).append('.').append(methodName).toString()
```

Comment:
Interessting. I just read this article: https://dzone.com/articles/string-concatenation-performacne-improvement-in-ja

Apparently, java will compile simple `+` operations into a stringbuilder, so, you also could use just `clazz.getName() + '.' + methodName` here if you want.

Reply:
Well the java compiler will, however javassist has its own compiler implementation. I'm not sure if they also implement this optimization, therefore I will leave it as Stringbuilder for now